### DISCUSSION TITLE
value comparison

### DISCUSSION THREAD

**Discussion by trend-jason-x-lin** [SigmaHQ/sigma-specification#62] [Q&A] [answered]
Hi all,

Is there a way to compare column values with specific values such as EventId > 10?
I saw there is an aggregation with the comparison (count(UserName) by SourceWorkstation > 3) but I don't need aggregation.
Thanks!

**Comment by trend-jason-x-lin** [comment]
Duplicate with discussion https://github.com/SigmaHQ/sigma/discussions/4035 here.

**Comment by thomaspatzke** [comment, marked as answer]
Yes, the very basic version of a detection item compares to a specific value: `EventID: 10`. Modifiers can be used to add numeric conditions, e.g. `EventID|gt: 10` does exactly what you describe above.
